packages/block-library/src/comment-template/util.js

```diff
@@ -0,0 +1,55 @@
+/**
+ *
+ * This function converts a flat list of comment objects with a `parent` property
+ * to a nested list of comment objects with a `children` property. The `children`
+ * property is itself a list of comment objects.
+ *
+ * @example
+ * ```
+ * const comments = [
+ *		{ id: 1, parent: 0 },
+ *		{ id: 2, parent: 1 },
+ *		{ id: 3, parent: 2 },
+ *		{ id: 4, parent: 1 },
+ *	]
+ * expect( convertToTree( comments ) ).toEqual( [
+ *	{
+ *		id: 1,
```

Comment:
This example is outdated because it contains the removed `id` field.
```suggestion
 *		id: `1,`
```